packages/@react-spectrum/table/stories/Table.stories.tsx

```diff
@@ -1305,7 +1306,7 @@ storiesOf('TableView', module)
   .add(
     'allowsResizing, onColumnResize action',
```

Comment:
Minor behavior nitpicks, feel free to discuss/punt/reject. I personally don't think they are super important/blockers but worth bringing up in case we haven't before (maybe we have already?):
1. Should/Can we have the table scroll as the user drags the resizer past the end of the table? At the moment, the user has to drag the column resizer past the end of the table -> manually scroll to the end of the table -> repeat the resizer drag operation. This is only really problematic if the table is at the edge of the window itself really though.

2. I wonder if it may be appropriate to scroll the column header into view on click if it is partially scrolled out of view (maybe something I could consider doing in the scroll into view PRs). It would avoid cases like this where the dropdown is positioned off the table.:
<img width="637" alt="image" src="https://user-images.githubusercontent.com/9661127/206008290-11ce1bca-7f02-4f85-b605-fbf21da1e58a.png">

Reply:
1. we've decided we only scroll the table when in keyboard mode, it shouldn't scroll when using the mouse or touch.
2. that's a good bug catch, might need to address it later

Reply:
TODO: add to followup